app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/components/PlanWizardVMStepTable.js

```diff
@@ -21,9 +21,9 @@ import {
   PAGINATION_VIEW
 } from 'patternfly-react';
 
-import rowFilter from './rowFilter';
-import searchFilter from './searchFilter';
-import CustomToolbarFind from './CustomToolbarFind';
+import rowFilter from '../../common/rowFilter';
```

Comment:
excellent! wonderful hoist here 👍 

though maybe the sortFilter listFilter or paginate could be as well?  Unclear if these can be used in this work.. will look at deeper on pass 2